test/test_nn.py

```diff
@@ -11662,6 +11662,8 @@ def check_lengths(lengths, enforce_sorted, use_default_hiddens, proj_size):
                 prec = dtype2prec_DONTUSE[dtype]
                 if dtype == torch.float16:
                     prec = 4e-2
+                elif dtype == torch.float32:
+                    prec = 2e-5
```

Comment:
Is this due to TF32? If e.g., `NVIDIA_TF32_OVERRIDE=0` causes the test to pass it might be more robust to use the `tf32_on_and_off` decorator here

Reply:
The issue is the TF32 decorator doesn't work because we are pulling the values out of this directly. The method already has the decorator @eqy